packages/core/src/index.ts

```diff
@@ -30,6 +30,7 @@ export {
     processorWarnings,
     validateScopingSelector,
     parsePseudoImport,
+    parseStGlobalCustomProperty,
```

Comment:
I know this is used outside the core, but I would prefer it being duplicated in the codemode and not adding another export for a deprecated feature (the duplicate code in the codemod would be kept long after the core would delete this code)